python/README.md

```diff
@@ -41,7 +41,7 @@ Quantum with all its dependencies:
 conda create -y -n cuda-quantum python=3.10 pip
 conda install -y -n cuda-quantum -c "nvidia/label/cuda-11.8.0" cuda
 conda install -y -n cuda-quantum -c conda-forge mpi4py openmpi cxx-compiler
-conda env config vars set -n cuda-quantum LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CONDA_PREFIX/envs/cuda-quantum/lib"
+conda env config vars set -n cuda-quantum LD_LIBRARY_PATH="$CONDA_PREFIX/envs/cuda-quantum/lib:$LD_LIBRARY_PATH"
```

Comment:
This is really nerve-wrecking. Conda has explicitly suggested users to **NOT** include conda paths in `LD_LIBRARY_PATH` (see, e.g., [here](https://docs.conda.io/projects/conda-build/en/stable/resources/use-shared-libraries.html#shared-libraries-in-macos-and-linux)). Given that NVIDIA is fully committed to make conda-forge a first-class citizen (ex: we attempt to do Day 1+N release on conda-forge for new CTKs, see e.g. [here](https://github.com/conda-forge/cuda-feedstock/issues/23)), may I nudge the team to put serious consideration to sticking to a safer practice for conda on your roadmap?

Reply:
One thing to point out here is that this only modifies LD_LIBRARY_PATH *within this conda environment*, not generally for the shell or session.